apps/spark/src/main/java/com/linkedin/openhouse/jobs/client/TablesClient.java

```diff
@@ -38,7 +38,7 @@
 @Slf4j
 @AllArgsConstructor
 public class TablesClient {
-  private static final int REQUEST_TIMEOUT_SECONDS = 60;
+  private static final int REQUEST_TIMEOUT_SECONDS = 180;
```

Comment:
Should we make slightly less than gateway timeout which is also 180?

Reply:
Will start with 180. We can tune it back if needed.